projects/errors/docs/format.md

```diff
@@ -0,0 +1,100 @@
+# Response format & examples
+
+Error responses from publiq APIs follow the [RFC7807](https://datatracker.ietf.org/doc/html/rfc7807) standard, also known as "Problem Details for HTTP APIs".
```

Comment:
Is it clear for client developers what "publiq APIs" are?

Maybe we need an extra section somewhere else (maybe on the doc home page) to clarify this. Or maybe we can clarify it here.

Reply:
This was just moved from existing documentation to a new page, and we use this in other places as well. I think it should become clear once we have "publiq platform", as well as from the URL (docs.publiq.be) and the logo in the top left (which mentions "publiq").

Reply:
But the home page will be expanded before we release publiq platform / these docs (to completely replace documentatie.uitdatabank.be) - so that can also further clarify it yes